Source/gg2/Objects/InGameElements/Spectator.events/End Step.xml

```diff
@@ -77,6 +77,11 @@ else if(tracking)
         }
         else
             stopTracking = true;
+        if (player.corpse and !player.object and tracking)
```

Comment:
Meep, no points for you either. It should be `instance_exists(player.corpse)`. I provoked the error by setting respawn time to 20 seconds.